experiments

```diff
@@ -1 +1 @@
-use-npm-ci=5
\ No newline at end of file
+use-npm-ci=0
```

Comment:
Do we care about trailing newline?

Reply:
Nope :) This is read programmatically and that was addressed in https://github.com/heroku/heroku-buildpack-nodejs/pull/637